sys-auth/nss-pam-ldapd/files/nss-pam-ldapd-0.9.12-netdb-defines.patch

```diff
@@ -0,0 +1,26 @@
+diff -uNr a/nss/hosts.c b/nss/hosts.c
```

Comment:
Please add bug references to any relevant Gentoo bugs to the top of the patch. Also could you run `scrub-patch` from `iwdevtools` on it?